plugin.cpp

```diff
@@ -87,45 +87,54 @@ static const char * def_cfg = QUOTE({
 		"type" : "string",
 		"displayName" : "Subject",
 		"order" : "9",
-		"default" : "Fledge alert notification sent by $NOTIFICATION_INSTANCE_NAME$ "
+		"default" : "Fledge alert notification sent by $NOTIFICATION_INSTANCE_NAME$ ",
+		"group" : "Message"
 		},
+	"email_body" : {
+		"description" : "The email body",
+		"type" : "string",
+		"displayName" : "Body",
+		"order" : "10",
+		"default" : "Notification alert",
+		"group" : "Message"
```

Comment:
It is probably better to rename the default group "Default Configuration" to "Email recipients/addresses/config".